mne/io/base.py

```diff
@@ -940,11 +941,15 @@ def resample(self, sfreq, npad=100, window='boxcar',
                      raw data effectively jitters trigger timings. It is
                      generally recommended not to epoch downsampled data,
                      but instead epoch and then downsample, as epoching
-                     downsampled data jitters triggers. See here for an
-                     example:
+                     downsampled data jitters triggers.
+                     See here for an example:
 
                          https://gist.github.com/Eric89GXL/01642cb3789992fbca59
 
+                     Aternatively, events can be constructed using the original
+                     data and jointly resampled with the raw data using the
+                     'events' parameter.
```

Comment:
Sorry to be a pest about this, but saying `Alternatively` after the warning makes it sound like this method will fix the issue, but it won't. So please move elsewhere (and fix `Aternatively` spelling)